build.gradle

```diff
@@ -53,6 +54,14 @@ dependencies {
 
     codeMetrics(group: 'org.terasology.config', name: 'codemetrics', version: '1.3.2', ext: 'zip')
 
+    // GLFW splash screen
+    implementation platform("org.lwjgl:lwjgl-bom:3.2.3")
+    implementation 'org.lwjgl:lwjgl'
+    implementation 'org.lwjgl:lwjgl-opengl'
+    implementation 'org.lwjgl:lwjgl-glfw'
```

Comment:
Is this using the version from the `platform(...)` directive above? Do you think it would help to have a dedicated `LWJGL_VERSION` constant here and explicitly use that for all LWJGL dependencies?

```
	 LWJGL_VERSION = "3.2.3"
	 implementation platform("org.lwjgl:lwjgl-bom:" + LWJGL_VERSION)
    implementation 'org.lwjgl:lwjgl' + LWJGL_VERSION
	 ...
```

Reply:
That'd be a @DarkWeird question, there's some new magic here and there like this. The separate var was the old approach, but maybe the versioned `bom` is a suggested new approach?